test/RelayServer.webpack.test.ts

```diff
@@ -7,8 +7,11 @@ describe('RelayServer-webpack', () => {
   before('create webpack', function () {
     this.timeout(15000)
     const jsrelayDir = path.join(__dirname, '..', 'jsrelay')
+    console.log('jsrelayDir=', jsrelayDir)
+    // const webpackEntry = require( jsrelayDir+'/webpack.config').entry
+    // assert.ok( fs.existsSync(jsrelayDir+'/'+webpackEntry), 'missing entry file: '+webpackEntry)
     fs.rmdirSync(path.join(jsrelayDir, 'dist'), { recursive: true })
-    childProcess.execSync('npx webpack', { cwd: jsrelayDir })
+    childProcess.execSync('sh -c "npx webpack"', { cwd: jsrelayDir, stdio: 'inherit' })
```

Comment:
What is even going on here? How come this file needs changes at all?

Reply:
removed (was historical reason: change in dpeloy script caused this to fail, so I added check code, including file listings.
then found a better way to debug: in circleci, you can launch with ssh: after a [failed] launch, you can ssh into the build machine to look around - which is much better than repeatedly pushing build script with debugging stuff...